### PR TITLE
Issue1325 @RedisTransactional

### PR DESCRIPTION
**Pull request by uweschaefer**

### CODE REVIEW

File: factcast-factus-redis/src/main/java/org/factcast/factus/redis/RedisWriterToken.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          public void run() {
+            liveness.set(lock.isLocked());
+          }
+        };
+    timer.scheduleAtFixedRate(timerTask, 0, (long) (watchDogTimeout / 1.5));
+  }
+
+  @Override
+  public void close() throws Exception {
+    lock.unlock();
```
Comment (otbe) [inline factcast-factus-redis/src/main/java/org/factcast/factus/redis/RedisWriterToken.java:48]: Please swap lines as discuseed :)

File: factcast-factus-redis/src/main/java/org/factcast/factus/redis/tx/RedisTransactionalLens.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        redissonTxManager.startOrJoin();
+        return redissonTxManager.getCurrentTransaction();
+      };
+    }
+    return null;
+  }
+
+  @Override
+  public void doClear() {
+    if (redissonTxManager.inTransaction()) {
```
Comment (otbe) [inline factcast-factus-redis/src/main/java/org/factcast/factus/redis/tx/RedisTransactionalLens.java:95]: Check is already done within `txManager` (rollback is a noop if no currentTx) 🤔
Same for commit

File: factcast-factus-redis/src/main/java/org/factcast/factus/redis/batch/RedisBatchedLens.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        RBatch rBatch = batchMan.getCurrentBatch();
+        return rBatch;
+      };
+    }
+    return null;
+  }
+
+  @Override
+  protected void doClear() {
+    if (batchMan.inBatch()) {
```
Comment (otbe) [inline factcast-factus-redis/src/main/java/org/factcast/factus/redis/batch/RedisBatchedLens.java:81]: Same as in txManager.

**Review by otbe — APPROVED** [review]

File: factcast-client-grpc/src/main/java/org/factcast/client/grpc/ClientStreamObserver.java [hunk 4]
```diff
@@ -46,7 +46,6 @@
 
   @Override
   public void onNext(MSG_Notification f) {
-    log.trace("observer got msg: {}", f);
```
[the remark this answers is not in the document]
Comment (uweschaefer) [reply factcast-client-grpc/src/main/java/org/factcast/client/grpc/ClientStreamObserver.java:49]: yes. pushing out ALL of the event batch is a bit too much for our dear logging infra

[the remark this answers is not in the document]
Comment (uweschaefer) [reply factcast-client-grpc/src/main/java/org/factcast/client/grpc/ClientStreamObserver.java:49]: and also a problem in terms of data sec.

**Pull request merged**